bases/br_anatel_banda_larga_fixa/backhaul/table_config.yaml

```diff
@@ -0,0 +1,156 @@
+source_bucket_name: br_anatel
+project_id_staging: input-dados
+project_id_prod: input-dados
+table_id: backhaul # AUTO GENERATED
+dataset_id: br_anatel_banda_larga_fixa # AUTO GENERATED
+
+url_ckan: https://basedosdados.org/dataset/br-anatel-banda-larga-fixa # AUTO GENERATED
+url_github: https://github.com/basedosdados/mais/tree/master/bases/br_anatel_banda_larga_fixa # AUTO GENERATED
+
+version: <v1.0> # REQUIRED
+
+last_updated: 2021-06-10 # AUTO GENERATED
+
+# Descreva a tabela. Essas são as primeiras frases que um usuário vai ver.
+# Você não precisa ser muito conciso. Sinta-se a vontade para dar exemplos de
+# como usar os dados.
+# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.
+description: |  # REQUIRED
+    Dados disponibilizados pela Lei de Acesso em Informação com dados da evolução ao longo dos anos da instalação de infraestrutura ligada ao plano de expansão da banda larga fixa no país desde a implementação do Programa Banda Larga nas Escolas (Decreto nº 6.424/2008)
+
+# Quem está completando esse arquivo config?
+published_by:
+    name: Lucas Nascimento Moreira  # REQUIRED
+    code_url: https://github.com/basedosdados/mais/tree/master/bases/br_anatel_banda_larga_fixa/code # REQUIRED
+    website: <website>
+    email: lucsas.nasc.m@gmail.com
+
+# Qual organização/departamento/pessoa tratou os dados?
+# As vezes há um ponto intermediário entre os dados originais e subir na Base dos Dados.
+# Se essa pessoa é você, preencha abaixo com suas informações.
+treated_by:
+    name: Gerência de Universalização e Ampliação do Acesso
+    code_url: <onde encontrar código de tratamento>
+    website: https://www.gov.br/anatel/pt-br/regulado/universalizacao/plano-nacional-de-banda-larga
+    email: pruv@anatel.gov.br
+
+# Se houve passos de tratamento, limpeza e manipulação de dados, descreva-os aqui.
+treatment_description: | 
+    Os dados foram padronizados sob as diretrizes da BD+ e para os municípios que já possuíam infraestrutura instalada antes de 2008 com o início da expansão, foram substituídos os astericos presente nas demais colunas para uma melhor compatibilização dos dados
+

```

Comment:
Pode transformar em bullet point.

- Padronização sob as diretrizes da BD+  para os municípios que já possuíam infraestrutura instalada antes de 2008 com o início da expansão

- Substituição dos asteriscos presente nas demais colunas para uma melhor compatibilização dos dados